src/Microsoft.DotNet.Cli.Utils/PathUtility.cs

```diff
@@ -91,6 +91,18 @@ public static void EnsureDirectory(string directoryPath)
         /// <summary>
         /// Returns path2 relative to path1, with Path.DirectorySeparatorChar as separator
         /// </summary>
+        public static string GetRelativePath(DirectoryInfo directory, FileSystemInfo childItem)
```

Comment:
This should just become a DirectoryInfo extension method [directoryInfo.GetRelativePathTo(new FileInfo(..))